Dockerfile

```diff
@@ -2,17 +2,17 @@ FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS
 WORKDIR /go/src/github.com/openshift/cloud-credential-operator
 COPY . .
 ENV GO_PACKAGE github.com/openshift/cloud-credential-operator
+ARG TARGETPLATFORM
 RUN go build -ldflags "-X $GO_PACKAGE/pkg/version.versionFromGit=$(git describe --long --tags --abbrev=7 --match 'v[0-9]*')" ./cmd/cloud-credential-operator
 RUN go build -ldflags "-X $GO_PACKAGE/pkg/version.versionFromGit=$(git describe --long --tags --abbrev=7 --match 'v[0-9]*')" ./cmd/ccoctl
-RUN make cross-build-darwin-amd64
-RUN make cross-build-windows-amd64
+RUN mkdir -p /go/src/${GO_PACKAGE}/_output/bin
+RUN if [[ "$TARGETPLATFORM" == "linux/amd64" ]] ; then make cross-build-darwin-amd64 cross-build-windows-amd64 ; fi
 
 FROM registry.ci.openshift.org/ocp/4.10:base
 COPY --from=builder /go/src/github.com/openshift/cloud-credential-operator/cloud-credential-operator /usr/bin/
 COPY --from=builder /go/src/github.com/openshift/cloud-credential-operator/ccoctl /usr/bin/
 COPY manifests /manifests
-COPY --from=builder /go/src/github.com/openshift/cloud-credential-operator/_output/bin/darwin_amd64/ccoctl /ccoctl/darwin_amd64/
-COPY --from=builder /go/src/github.com/openshift/cloud-credential-operator/_output/bin/windows_amd64/ccoctl.exe /ccoctl/windows_amd64/
+COPY --from=builder /go/src/github.com/openshift/cloud-credential-operator/_output/bin/ /ccoctl/
```

Comment:
Do we have linux binary in /usr/bin and mac/windows binary in /ccoctl/ ?

Reply:
Yes. The linux binaries for all architectures will live in /usr/bin (but of course to get the arm64 linux binary you need to extract from an arm64 container image)
We just need a place to put these non-native binaries so that other tooling can extract and then sign them.